src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -858,48 +859,19 @@ public void canCheckOutInProcessItemWithRequest() {
   }
 
   @Test
-  public void checkOutFailsWhenCirculationRulesReferenceInvalidLoanPolicyId() {
-    setInvalidLoanPolicyReferenceInRules("some-loan-policy");
-
-    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
-    final IndividualResource steve = usersFixture.steve();
-
-    final DateTime loanDate = new DateTime(2018, 3, 18, 11, 43, 54, UTC);
-
-    final Response response = loansFixture.attemptCheckOutByBarcode(500,
-      new CheckOutByBarcodeRequestBuilder()
-        .forItem(smallAngryPlanet)
-        .to(steve)
-        .on(loanDate)
-        .at(servicePointsFixture.cd1()));
-
-    assertThat(response.getBody(),
-      is("Loan policy some-loan-policy could not be found, please check circulation rules"));
-
-    smallAngryPlanet = itemsClient.get(smallAngryPlanet);
-
-    assertThat(smallAngryPlanet, hasItemStatus(AVAILABLE));
-  }
-
-  @Test
-  public void checkOutDoesNotFailWhenCirculationRulesReferenceInvalidNoticePolicyId() {
```

Comment:
Similarly to my other comments, I think the circumstances under which this can occur have changed, not that this is no longer a possibility

Reply:
The test have been returned back and updated

Reply:
Thank you